docker/deployment/app-deployment.yaml

```diff
@@ -178,3 +178,37 @@ spec:
               secretKeyRef:
                 key: LD_TRUST_TOKEN
                 name: ld-trust-token
+      - name: lfmerge
+        # TODO: Decide on a verion-numbering policy for LfMerge releases
+        image: ghcr.io/sillsdev/lfmerge:latest
```

Comment:
excellent!  How did you like ghcr as a repo?

Reply:
Very easy to use from GHA workflows. And the fact that you can delete packages with less than 1,000 downloads means I don't have to worry about cluttering up the repo with failed builds if I end up creating any, which is nice.